lib/components/admin/query-record.js

```diff
@@ -13,14 +14,22 @@ import {CallRecordButton, CallRecordIcon} from './styled'
 class QueryRecordLayout extends Component {
   _getParams = () => {
     const {query} = this.props
-    return planParamsToQuery(JSON.parse(query.queryParams))
+    const originalQueryParams = JSON.parse(
+      query.queryParams,
+      // convert boolean for the 'arriveBy' field to strings
+      // per https://github.com/opentripplanner/otp-ui/blob/master/packages/core-utils/src/query.js#L285
+      // FIXME: move this elsewhere?
+      (key, value) => key === 'arriveBy'
```

Comment:
@binh-dam-ibigroup, I think this needs to become a fiield-trip util because it is also used here: https://github.com/opentripplanner/otp-react-redux/blob/master/lib/actions/field-trip.js#L217

Reply:
Thanks for pointing that out, code updated in https://github.com/opentripplanner/otp-react-redux/pull/395/commits/a2cfb7341f39d940744a3c02d4c637193df6eb97.